contracts/AuctionHouse.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.3;
+
+// TODO: Use interface. The current version of @openZeppelin/contracts lib has none that fits.
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "./auctions/FungibleAuction.sol";
+
+/// @title Auction House
+contract AuctionHouse {
+    /// @dev All fungible auctions that are either starting in the future, already active,
+    /// or haven't had their tokens withdrawn yet after finishing.
+    address[] public fungibleAuctions;
```

Comment:
Consider swap this with an OZ enumerable set, it might save you some code

Reply:
Considered, and accepted. Might also be useful later on. Will change!